### PR TITLE
Feature/self serve generated travel certificate/merged

### PR DESCRIPTION
**Pull request by sladan-hedvig**
## [APP-XXX]

- Added travel certificate

## Checklist

- [ ] Dark/light mode verification
- [ ] Landscape verification
- [ ] iPad verification
- [ ] iPod/iPhone 12 mini/iPhone SE verification

### CODE REVIEW

File: Projects/Claims/Sources/CommonClaims/BulletPointCard.swift [hunk 1]
```diff
@@ -28,10 +28,9 @@ extension BulletPointCard: Reusable {
         contentView.layoutMargins = UIEdgeInsets(top: 20, left: 10, bottom: 20, right: 20)
         contentView.isLayoutMarginsRelativeArrangement = true
         contentView.insetsLayoutMarginsFromSafeArea = false
-
         cardContainer.addSubview(contentView)
-
-        contentView.snp.makeConstraints { make in make.leading.equalToSuperview().inset(40)
+        contentView.snp.makeConstraints { make in
```
Comment (julia-andersson) [inline Projects/Claims/Sources/CommonClaims/BulletPointCard.swift:32]: should this file really be in claims or can we move it to hCoreUI?

Comment (julia-andersson) [inline Projects/Claims/Sources/CommonClaims/BulletPointCard.swift:32]: should this file really be in claims or can we move it to hCoreUI?

File: Projects/Contracts/Sources/Screens/TravelInsuranceFlowJourney.swift [hunk 2]
```diff
@@ -0,0 +1,182 @@
+import Foundation
+import Presentation
+import SwiftUI
```
Comment (julia-andersson) [inline Projects/Contracts/Sources/Screens/TravelInsuranceFlowJourney.swift:3]: Rename "screens" folder to "Journeys"

**Review by julia-andersson — CHANGES_REQUESTED** [review]

File: Projects/Claims/Sources/CommonClaims/BulletPointCard.swift (hunk 1 above)
Comment (sladan-hedvig) [reply Projects/Claims/Sources/CommonClaims/BulletPointCard.swift:32]: I think we are going to have a major change of UI over the summer and lots of things will be moved around and I feel it is not just right moment to take care of that.

File: Projects/Contracts/Sources/Screens/TravelInsuranceFlowJourney.swift (hunk 2 above)
Comment (sladan-hedvig) [reply Projects/Contracts/Sources/Screens/TravelInsuranceFlowJourney.swift:3]: Yeah, we can do that

Comment (sladan-hedvig) [reply Projects/Contracts/Sources/Screens/TravelInsuranceFlowJourney.swift]: Done

**Pull request merged**